### ISSUE TITLE
Mute and Unmute 

### ISSUE THREAD

**Issue by kewalsk** [Pmmlabs/OpenPeriscope#30]
Did you consider to add the information if the user is muted (is_muted) and to add function buttons to mute/unmute?

**Comment by Pmmlabs** [comment]
@kewalsk, I don't plan to add mute/unmute functions for now.

**Comment by kewalsk** [comment]
Then I will do it myself. It seems to be pretty strightforward.

**Comment by Pmmlabs** [comment]
There will be not only API call, because notifications in OpenPeriscope are implemented by periodical checking new broadcasts. If you call `mute` API method, you will stop receiving push notifications on mobile devices, but not in OpenPeriscope. i.e. you need to store list of muted users, and to exclude them from `followingBroadcastFeed` response.

**Comment by kewalsk** [comment]
So the notifications in OpenPeriscope will be sent even if the user is muted from mobile device? Checking is_muted just before creating Notification object could be solution but it requires additional api call.

**Comment by Pmmlabs** [comment]
Yes, it's flaw, I initially wanted to implement it in the right way - using Push API (which is experimental technology and supported by newest Chrome and Firefox only), but didn't find out this, and used periodical ajax. Therefore can be some difficulties.

**Comment by kewalsk** [comment]
I don't have experience with push messaging but maybe [FCM](https://firebase.google.com/docs/cloud-messaging/) or [GCM](https://developers.google.com/cloud-messaging/) could be solution, or is this the same?

**Comment by Pmmlabs** [comment]
Yes, Periscope uses GCM but I don't know how to implement it on JS for a while.
